pkg/omf/cli/omf.destroy.fish

```diff
@@ -1,10 +1,19 @@
 function omf.destroy -d "Remove Oh My Fish"
   echo (omf::dim)"Removing Oh My Fish..."(omf::off)
 
-  omf.remove_package (basename $OMF_PATH/pkg/*) >/dev/null ^&1
+  for pkg in (basename $OMF_PATH/pkg/*)
+    omf.remove_package $pkg >/dev/null ^&1
+  end
+
+  set -l fish_config $XDG_CONFIG_HOME/fish
+  if test "$fish_config" = "/fish"
```

Comment:
Being picky, I'd add a path suffix, and also rename the next variable localconfig adding a underscore as a space replacement.